Framework/PythonInterface/core/src/Converters/CloneToNDArray.cpp

```diff
@@ -19,7 +19,20 @@
 #define NO_IMPORT_ARRAY
 #include <numpy/arrayobject.h>
 
-namespace Mantid::PythonInterface::Converters::Impl {
+namespace Mantid::PythonInterface::Converters {
+#ifdef __APPLE__
```

Comment:
I'm not sure why this is required on clang. The symbols should be exported correctly...